packages/react-app/src/components/field-input/text-field-input.tsx

```diff
@@ -124,6 +124,7 @@ export default function TextFieldInput({
                 component: 'a',
                 props: {
                   target: '_blank',
+                  tabindex: '-1',
```

Comment:
We will probably want the navigation for quest details but for now its good because this will be handled in this card : 
https://app.zenhub.com/workspaces/quests-6092dda4c272a5000e858266/issues/1hive/quests/210
We will not have anything interactable in the summary view of quest

Reply:
I see.